tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
-    "lib": ["es2017"],
+    "lib": ["es2017", "dom"],
```

Comment:
Why did we add this? AFAIK we shouldn't need to rely on any browser APIs.

Reply:
It's a requirement of the `create-require` module, unfortunately - it pulls in the "URL" type which is only included when dom is included, as far as I understand

Reply:
Ah okay. Node has exposed `URL` on the global object since v10 according to [the docs](https://nodejs.org/api/url.html#url_the_whatwg_url_api) so I wonder if we're missing some config to make TypeScript be happy with it?

Reply:
Fixed in https://github.com/nuxt-contrib/create-require/pull/3

Reply:
Oh actually looks like an outstanding issue, see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/34960.

I think we should try to avoid having "dom" in there if possible as it would make it easy for contributors to mistakenly try to use browser-only APIs. Maybe we could override the type declaration for `create-require` to make it:

```typescript
export default function createRequire (filename: string): NodeRequire;
```

(Which is how we're using it anyway.)

Reply:
Fix has been released

Reply:
Will push an update here in just a moment

Reply:
Done

Reply:
That was fast! :)